extraTs/utils/Validation.ts

```diff
@@ -1,30 +1,31 @@
-import validateEmail from './helpers'
-
-export default function validateUsers(users : IUsers[]) : void{
-    let countValid : number =0;
-    let countInvalid : number =0;
-    let a : string[] = [];
-    let b : string[] = [];
-    users.forEach(function(item){
-        const {traineeEmail, reviewerEmail} = item;
-        if(validateEmail(traineeEmail)){
-            a.push(traineeEmail);
-            countValid++;
-        }
-        else{
-            b.push(traineeEmail);
-            countInvalid++;
-        }
-        if(validateEmail(reviewerEmail)){
-            a.push(reviewerEmail);
-            countValid++;
-        }
-        else{
-            b.push(reviewerEmail);
-            countInvalid++;
-        }
-    })
-    console.log("Valid is :"+a);
-    console.log("invalid is :" +b);
-    console.log("Valid: "+countValid+" , Invalid: "+countInvalid);
-};
+import validateEmail from './helpers'
+export default function validateUsers(users){
+
+    let countValid=0;
+    let countInvalid=0;
+    let a =[];
+    let b =[];
+    users.forEach(function(item){
+        const {traineeEmail, reviewerEmail} = item;
+        if(validateEmail(traineeEmail)){
+            a.push(traineeEmail);
+            countValid++;
+        }
+        else{
+            b.push(traineeEmail);
+            countInvalid++;
+        }
+        if(validateEmail(reviewerEmail)){
+            a.push(reviewerEmail);
+            countValid++;
+        }
+        else{
+            b.push(reviewerEmail);
+            countInvalid++;
+        }
+    })
+    console.log("Valid is :"+a);
+    console.log("invalid is :" +b);
+    console.log("Valid: "+countValid+" , Invalid: "+countInvalid);
+};
+
```

Comment:
use es6, proper function naming and define type for parameters

Reply:
used ES6 , proper naming function as well as defined parameters.